Apps/Playground/Scripts/experience.js

```diff
@@ -53,7 +53,7 @@ function CreateInputHandling(scene) {
 
 _native.whenGraphicsReady().then(function () {
 
-    var engine = new BABYLON.NativeEngine();
+    var engine = new BABYLON.NativeEngine(true);
```

Comment:
What is this extra param? When I look at `NativeEngine` in Babylon.js I don't see a constructor that takes a parameter.

Reply:
I will be adding an additional parameter to the NativeEngine constructor for an `adaptToDeviceRatio` flag, similar to the parameter used for the ThinEngine.

Reply:
A bool is kind of confusing here. Would it make sense to take a configuration object instead so you could do something like:

`var engine = new BABYLON.NativeEngine({ adaptToDeviceRatio: true });`

Reply:
On the one hand, we might want to match the ThinEngine constructor as closely as possible, on the other hand, the constructor already uses things like HTMLCanvasElement that may not be all that useful for us...

Sure, a configuration object might be the most maintainable solution, since we can add more recognized members of this object as we add more options.

Reply:
Opened https://github.com/BabylonJS/Babylon.js/pull/9892 to add this to Babylon.js's constructor